nnsvs/bin/generate.py

```diff
@@ -47,16 +48,37 @@ def my_app(config : DictConfig) -> None:
     with torch.no_grad():
         for idx in tqdm(range(len(in_feats))):
             feats = torch.from_numpy(in_feats[idx]).unsqueeze(0).to(device)
-            out = model(feats, [feats.shape[1]]).squeeze(0).cpu().data.numpy()
 
-            out = scaler.inverse_transform(out)
-
-            # Apply MLPG if necessary
-            if np.any(model_config.has_dynamic_features):
-                windows = get_windows(3)
-                out = multi_stream_mlpg(
-                    out, scaler.var_, windows, model_config.stream_sizes,
-                    model_config.has_dynamic_features)
+            if model.prediction_type == "probabilistic":
```

Comment:
Please consider changing this like:

`if not model.is_deterministic()` or `if model.prediction_type() == ENUM_VALUE`

Reply:
I changed to use  predictin_type() of BaseModel in the latest commit.